src/store/utility.ts

```diff
@@ -92,6 +92,18 @@ function replaceTag(
   return result;
 }
 
+export function textExportToText(targettext: string): string {
+  let resolvedText = targettext.replace(/\{.*?\}/g, "");
+  resolvedText = resolvedText.replace(/\[(.*?)\]/g, "$1");
```

Comment:
この処理、少し意図してないものとなっている気がします。
例えば、かなり特殊な例ではありますが、`{ここにテキストを入力}と表示されている場所に...`といった入力がされた場合、`{ここにテキストを入力}`の部分が消し飛ぶだけになりうる処理になっています。

なので、以下のように一発で置換する正規表現が正しい処理になると思います...!
```suggestion
  let resolvedText = targettext.replace(/\{.*?\}\[(.*?)\]/g, "$1");
```

Reply:
レビューありがとうございます。参考になりました。下記のコードだとうまくいくようです。
```
const resolvedText = targettext.replace(/\[(.*?)\]\{.*?\}/g, "$1");
```

![image](https://github.com/VOICEVOX/voicevox/assets/100256521/ef5f496a-bb8c-4f49-9da5-3da79f68efea)
![image](https://github.com/VOICEVOX/voicevox/assets/100256521/c7a918fc-b241-44a4-b358-feddc9b9308d)

音声の書き出しも同様に修正します。

Reply:
> {ここにテキストを入力}と表示されている場所に...といった入力がされた場合、{ここにテキストを入力}の部分が消し飛ぶだけになりうる処理になっています

あ、この挙動を想定していました！
目的は字幕で消えるテキストを書くことなので、これで良いかな〜と思ってました。

例えば、たまにVOICEVOXのテクニックで最初に「っ」を入れると発音が正しくなるとか、語尾の調整のために伸ばし棒を大量に置くみたいなことがあって、それをテキスト書き出しで無視するみたいな。

入力テキスト
`{っ}はい、わかりました{ーーー}`

出力テキスト
`はい、わかりました`

とか。